### ISSUE TITLE
<layer- style="opacity: 0.5"> does not set initial opacity

### ISSUE THREAD

**Issue by prushforth** [Maps4HTML/Web-Map-Custom-Element#667]
According to the spec, the \<layer-> element should have a settable opacity, but it doesn't currently work.

> [layer](https://maps4html.org/MapML/spec/#the-layer-element) transparency is controlled by the CSS opacity property. [[SVG2](https://maps4html.org/MapML/spec/#bib-svg2)] [[css-color-3](https://maps4html.org/MapML/spec/#bib-css-color-3)]
